### PR TITLE
Add cpuid_count function

### PR DESCRIPTION
**Pull request by PflaeginGmbh**
Hello there,
I came across this package and find it very useful. One feature I was missing was the ability to access subleafs with cpuid. So I added this functionality.
I would be happy if my changes make the package even more useful.

### CODE REVIEW

File: src/cpuid/bindings.pyi [hunk 1]
```diff
@@ -1,4 +1,5 @@
-from typing import Tuple
+def cpuid(level: int) -> tuple[int, int, int, int]:
```
Comment (odesenfans) [inline src/cpuid/bindings.pyi]:
```suggestion
def cpuid(level: int) -> Tuple[int, int, int, int]:
```

Use the type from the typing module, this is breaking for older versions of Python.

File: src/cpuid/bindings.pyi [hunk 2]
```diff
@@ -1,4 +1,5 @@
...
+    ...
 
-def cpuid(level: int) -> Tuple[int, int, int, int]:
+def cpuid_count(level: int, count: int) -> tuple[int, int, int, int]:
```
Comment (odesenfans) [inline src/cpuid/bindings.pyi]: Same, use `Tuple`.

**Review by odesenfans — CHANGES_REQUESTED** [review]
Looks good, just a minor comment on type hints.

File: src/cpuid/bindings.pyi (hunk 1 above)
Comment (PflaeginGmbh) [reply src/cpuid/bindings.pyi]: You're right, done in [08145e0](https://github.com/bearmetaldev/python-cpuid/pull/8/commits/08145e064176bbfdfedf16aa60902d51826f308f).

**Review by odesenfans — APPROVED** [review]

**Pull request merged**